internal/settings/providers/allinkl/provider.go

```diff
@@ -0,0 +1,179 @@
+package allinkl
```

Comment:
Please remove the binary file `internal/settings/providers/.DS_Store`

Reply:
done.